source/guide_seafile.rst

```diff
@@ -0,0 +1,271 @@
+ [isabell@stardust ~]$ cd seafile/
+ [isabell@stardust ~]$ tar -xzf seafile-server_*
+ [isabell@stardust ~]$ mkdir installed
+ [isabell@stardust ~]$ mv seafile-server_* installed/
+ [isabell@stardust ~]$
+
+Step 2
+------
+
+Install the required python libraries.
+
+::
+
+ [isabell@stardust ~]$ curl --silent http://effbot.org/media/downloads/Imaging-1.1.7.tar.gz | tar -xzf -
+ [isabell@stardust ~]$ cd Imaging-1.1.7
+ [isabell@stardust ~]$ perl -pi -e 's|/usr/lib|/usr/lib64|g' setup.py
+ [isabell@stardust ~]$ python2.7 setup.py install --user
+ [isabell@stardust ~]$ cd ..
+ [isabell@stardust ~]$ rm -rf Imaging-1.1.7
+ [isabell@stardust ~]$
+
+Step 3
+------
+
+Create the needed MySQL databases as the installer file won't work on uberspace.
+
+.. warning:: Replace ``isabell`` with your username!
+
+::
+
+ [isabell@stardust ~]$ mysql
+ MariaDB [(none)]>     create database `isabell_ccnet-db` character set = 'utf8';
+ MariaDB [(none)]>     create database `isabell_seafile-db` character set = 'utf8';
+ MariaDB [(none)]>     create database `isabell_seahub-db` character set = 'utf8';
+ MariaDB [(none)]>     exit;
+ [isabell@stardust ~]$
+
+Step 4
+------
+
+Run the seafile installer script.
+
+::
+
+ [isabell@stardust ~]$ cd ~/seafile/seafile-server-*
+ [isabell@stardust ~]$ ./setup-seafile-mysql.sh
+ [isabell@stardust ~]$
+
+Important answers:
+
+.. warning:: Replace ``isabell`` with your username!
+
+.. code-block:: console
+  :emphasize-lines: 8,11,14,19,24
+
+  -------------------------------------------------------
+  Please choose a way to initialize seafile databases:
+  -------------------------------------------------------
+
+  [1] Create new ccnet/seafile/seahub databases
+  [2] Use existing ccnet/seafile/seahub databases
+
+  [ 1 or 2 ] 2
+
+  Which mysql user to use for seafile?
+  [ mysql user for seafile ] isabell
+
+  Enter the existing database name for ccnet:
+  [ ccnet database ] isabell_ccnet-db
+
+  verifying user "isabell" access to database isabell_ccnet-db ...  done
+
+  Enter the existing database name for seafile:
+  [ seafile database ] isabell_seafile-db
+
+  verifying user "isabell" access to database isabell_seafile-db ...  done
+
+  Enter the existing database name for seahub:
+  [ seahub database ] isabell_seahub-db
+
+  verifying user "isabell" access to database isabell_seahub-db ...  done
+
+Configure port
+--------------
+
+Since seafile uses its own webserver and fileserver, you need to find 2 free ports. Run this command 2 times and write down the 2 ports.
+
+.. include:: includes/generate-port.rst
+
+Step 5
+------
+
+Enter your domain name in config; Edit ``~/seafile/conf/ccnet.conf``
+
+.. warning:: Replace ``isabell`` with your username!
+
+.. code-block:: console
+  :emphasize-lines: 1
+
+  SERVICE_URL = https://isabell.uber.space/
+
+Step 6
+------
+
+Change seahub port in config; Edit ``~/seafile/conf/gunicorn.conf``
+
+.. warning:: Replace ``<yourport>`` with your first port!
```

Comment:
Please rename the two ports to `gunicorn-port` port and `fileserver-port` to make things clearer. Here, in "Configure port" further up and down in the htaccess